.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -1,26 +1,59 @@
 ## @file
-# Azure Pipeline build file for a build using ubuntu and GCC5
+# Azure Pipeline build file for a build using Ubuntu and GCC.
+#
+# IMPORTANT: This file requires the local repo to contain a .azurepipelines/Matrix-Build-Job.yml file that
+#            performs the actual build steps for the configurations and packages supported by the repo.
+#
+# NOTE: This file is automatically synchronized from Mu DevOps. Update the original file there
+#       instead of the file in this repo.
+#
+# - Mu DevOps Repo: https://github.com/microsoft/mu_devops
+# - File Sync Settings: https://github.com/microsoft/mu_devops/blob/main/.sync/Files.yml
 #
 # Copyright (c) Microsoft Corporation.
-# Copyright (c) 2020, Hewlett Packard Enterprise Development LP. All rights reserved.<BR>
-# Copyright (c) 2022, Loongson Technology Corporation Limited. All rights reserved.<BR>
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 ##
-trigger:
-- master
-- stable/*
-pr:
-- master
-- stable/*
 
 variables:
-  - template: templates/defaults.yml
+- group: architectures-arm-64-x86-64
+- group: tool-chain-ubuntu-gcc
+- group: coverage
 
-jobs:
-- template: templates/pr-gate-build-job.yml
+extends:
+  template: MuDevOpsWrapper.yml
   parameters:
-    tool_chain_tag: 'GCC5'
-    vm_image: 'ubuntu-22.04'
-    container: ${{ variables.default_linux_image }}
-    arch_list: "IA32,X64,ARM,AARCH64,RISCV64,LOONGARCH64"

```

Comment:
nit: does this really work? if so, should we keep them?

Reply:
You're specifically talking about all of the archs, right? I think that would be a larger change if we wanted to build those where we'd also need to build RISCV64 and LOONGARCH64 in other repos like Mu Basecore, Mu Tiano Plus, etc.

Historically, we haven't supported those. The changes needed across the entire codebase, keeping up with toolchains, and additional build resources is extra overhead and any issues that may occur on untested archs during upstream can be resolved in edk2 CI.

Probably something that would need more discussion.

Reply:
I marked this as resolved assuming this was bringing that to attention not something that needs to be addressed in this PR.